variables.tf

```diff
@@ -1,14 +1,3 @@
-
-## var scratchpad
-## required banking vars (generally provided by workspace)
-variable "app_name" {
-  type = string
-  description = "describe your variable"
-}
-variable "environment" {
```

Comment:
Without `environment` how are we getting the environment for function names? Or are we expecting users to handle the abstraction?

Reply:
`environment` in this context is not what I expected either. It's being used as a variable for environment variables. If you check the lambda/main.tf you'll find this:
```
dynamic "environment" {
    for_each = length(var.environment) < 1 ? [] : [var.environment]
    content {
      variables = environment.value.variables
    }
  }
```

How can we move away from that and give them a different way to send through envvars?